content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,385 @@
+3. Attest the SPIRE Agent running on the node
+   {{< text bash >}}
+   $ kubectl exec -n spire $SPIRE_SERVER_POD -- \
+   /opt/spire/bin/spire-server entry create \
+   -spiffeID spiffe://example.org/ns/spire/sa/spire-agent \
+   -selector k8s_psat:cluster:demo-cluster \
+   -selector k8s_psat:agent_ns:spire \
+   -selector k8s_psat:agent_sa:spire-agent \
+   -node -socketPath /run/spire/sockets/server.sock
+   
+   Entry ID         : d38c88d0-7d7a-4957-933c-361a0a3b039c
+   SPIFFE ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Parent ID        : spiffe://example.org/spire/server
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s_psat:agent_ns:spire
+   Selector         : k8s_psat:agent_sa:spire-agent
+   Selector         : k8s_psat:cluster:demo-cluster
+   {{</ text >}}
+
+4. and then register an entry for istio-ingressgateway pod.
+   {{< text bash >}}
+   $ kubectl exec -n spire $SPIRE_SERVER_POD -- \
+   /opt/spire/bin/spire-server entry create \
+   -spiffeID spiffe://example.org/ns/istio-system/sa/istio-ingressgateway-service-account \
+   -parentID spiffe://example.org/ns/spire/sa/spire-agent \
+   -selector k8s:sa:istio-ingressgateway-service-account \
+   -selector k8s:ns:istio-system \
+   -selector k8s:pod-uid:$INGRESS_POD_UID \
+   -dns $INGRESS_POD \
+   -dns istio-ingressgateway.istio-system.svc \
+   -socketPath /run/spire/sockets/server.sock
+
+   Entry ID         : 6f2fe370-5261-4361-ac36-10aae8d91ff7
+   SPIFFE ID        : spiffe://example.org/ns/istio-system/sa/istio-ingressgateway-service-account
+   Parent ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s:ns:istio-system
+   Selector         : k8s:pod-uid:63c2bbf5-a8b1-4b1f-ad64-f62ad2a69807
+   Selector         : k8s:sa:istio-ingressgateway-service-account
+   DNS name         : istio-ingressgateway.istio-system.svc
+   DNS name         : istio-ingressgateway-c48554dd6-cff5z
+   {{</ text >}}
+
+5. Deploy an example workload:
+
+{{< text bash >}}
+$ istioctl kube-inject --filename @samples/httpbin/httpbin.yaml | kubectl apply -f -
+{{< /text >}}
+
+6. Get pod information:
+
+{{< text bash >}}
+$ HTTPBIN_POD=$(kubectl get pod -l app=httpbin -o jsonpath="{.items[0].metadata.name}")
+$ HTTPBIN_POD_UID=$(kubectl get pods $HTTPBIN_POD -o jsonpath='{.metadata.uid}')
+{{< /text >}}
+
+7. Register workload.
+
+   {{< text bash >}}
+   $ kubectl exec -n spire spire-server-0 -- \
+   /opt/spire/bin/spire-server entry create \
+   -spiffeID spiffe://example.org/ns/default/sa/httpbin \
+   -parentID spiffe://example.org/ns/spire/sa/spire-agent \
+   -selector k8s:ns:default \
+   -selector k8s:pod-uid:$HTTPBIN_POD_UID \
+   -dns $HTTPBIN_POD \
+   -socketPath /run/spire/sockets/server.sock
+   {{< /text >}}
+
+{{< warning >}}
+SPIFFE IDs for workloads must follow the Istio SPIFFE ID pattern: spiffe://<trust.domain>/ns/<namespace>/sa/<service-account>

```

Comment:
can we use code snippets inside `{{<warning >}}`?
```suggestion
SPIFFE IDs for workloads must follow the Istio SPIFFE ID pattern: `spiffe://<trust.domain>/ns/<namespace>/sa/<service-account>`

```

Reply:
Let me check

Reply:
Yes, it is possible